runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -155,7 +155,10 @@
 	public static final int SAND_COLLECTION = 895;
 	public static final int PISCARILIUS_CRANE_REPAIR = 7199;
 	public static final int HOME_MAKE_TABLET = 4067;
-
+	public static final int THIEVING_STALL = 832;
+	public static final int PICKPOCKET_SUCCESS = 881;
+	public static final int PICKPOCKET_FAIL = 424;
```

Comment:
PICKPOCKET_FAIL is the same as BLOCK_UNARMED

Reply:
removed + added comment to block_unarmed